paasta_tools/firewall_update.py

```diff
@@ -95,16 +92,12 @@ def process_inotify_event(event, services_by_dependencies):
 
 
 def services_running_here(soa_dir):
-    # Generator helper that yields (service, instance, config) of both marathon and chronos tasks
-    cluster = load_system_paasta_config().get_cluster()
-
+    # Generator helper that yields (service, instance, framework) of both marathon and chronos tasks
```

Comment:
If you change what this returns you'll need to fix up its existing callers, like smartstack_dependencies_of_running_firewalled_services()